dias/analyzers/dataset_analyzer.py

```diff
@@ -80,7 +80,11 @@ def run(self):
         ds.get.index()
 
         # Use the tracker to get the chimestack files to analyze
-        cs_file_list = self.new_files(filetypes=self.instrument + "_corr")
+        end_time = datetime.datetime.utcnow() - self.offset
```

Comment:
This didn't do what I expected it to do. It still included the files from less than `offset` ago. I had to add an additional 8h to `offset`. Maybe there is an error with UTC vs local time in the tracker?

Reply:
The way the tracker currently works is that it pulls all `start_time < end`. That means if the file starts earlier than the provided `end` then it returns the file. This is more liberal than checking `end_time > end`, and if so, not returning the file. Could that be part of it?

Reply:
(Discussed over Slack)